doc/architecture/decisions/0009-primary-person.md

```diff
@@ -31,21 +27,30 @@ The fact store would be used for storage of items that are not direct answers bu
     "lists": [
         {
             "name": "people",
+            "context": {
+                "primary_person": "a9hd8j"
+            },
             "items": [
                 "a9hd8j"
             ]
         }
-    ],
-    "facts": {
-        "primary_person": {"list_item_id": "a9hd8j"}
-    }
+    ]
 }
 ```
 
-It is not necessary for a fact to be stored against anything other than a list item at the time of writing.
+### Create a PrimaryPersonCollector block type
+
+A new block type "PrimaryPersonCollector" will be defined to support the necessary behaviour. It will extend the existing ListCollector and support adding a primary person through adding context to the list store. The following diagram identifies the flow to be implemented in runner.  
+
+![Primary Flow](0009-primary-flow.png)
```

Comment:
i think the id in the url needs to be removed from `/do-you-live-at/<id>/add`?

Reply:
Does it? This is the suggested url for adding answers to an existing list item

Reply:
oh, is this the list id or the list item id?

Reply:
List item id - I can update the diagram to make things clearer though

Reply:
is this not inconsistent with the list collector add?

Reply:
The root part of the urls should be the list name, not the list collector block name. This was changed during the implementation of lists - https://github.com/ONSdigital/eq-survey-runner/blob/v3/doc/architecture/decisions/0005-simplify-urls-and-extend-to-support-repeating-effectively.md was updated with the new urls, but the images in https://github.com/ONSdigital/eq-survey-runner/blob/v3/doc/architecture/decisions/0006-make-named-lists-a-first-class-construct.md weren't.

Reply:
I think it would also be clearer if the url for the add primary person was `/<list_name>/<list_item_id>/primary-person` as it's both the add and edit screen.